opis-manager/src/app/home/home.component.ts

```diff
@@ -59,63 +70,74 @@ export class HomeComponent implements OnInit {
   constructor(
     public configService: ConfigService,
     private http: HttpClient
-    ) { }
+  ) { }
 
   ngOnInit() {
     this.configService.getConfig()
-    .subscribe((data: Config) => {
-      this.config = {
-        apiUrl: data.apiUrl,
-        years: data.years
-      };
+      .subscribe((data: Config) => {
+        this.config = {
+          apiUrl: data.apiUrl,
+          years: data.years
+        };
 
-      this.optionsSlider = {
-        floor: 1,
-        ceil: data.years.length,
-        showTicksValues: true,
-        getLegend: (value: number): string => {
-          return this.config.years[value - 1];
-        }
-      };
+        this.optionsSlider = {
+          floor: 1,
+          ceil: data.years.length,
+          showTicksValues: true,
+          getLegend: (value: number): string => {
+            return this.config.years[value - 1];
+          }
+        };
 
-      this.maxValue = this.config.years.length;
+        this.maxValue = this.config.years.length;
 
-      this.getDepartmnets();
-    });
+        this.getAllDepartments();
+      });
+
+    //ChartJS Annotation plugin stuff
+    let namedChartAnnotation = ChartAnnotation;
```

Comment:
I don't see any re-assign of this, so probably we can use **const** instead of **let**.